hazelcast/src/main/java/com/hazelcast/cp/internal/datastructures/spi/client/CreateRaftGroupMessageTask.java

```diff
@@ -38,7 +38,7 @@ public CreateRaftGroupMessageTask(ClientMessage clientMessage, Node node, Connec
     @Override
     protected void processMessage() {
         RaftService service = nodeEngine.getService(RaftService.SERVICE_NAME);
-        service.createRaftGroupForProxyAsync(parameters).whenCompleteAsync(this);
+        service.createRaftGroupForProxyAsync(parameters).whenCompleteAsync(this, internalAsyncExecutor);
```

Comment:
`CALLER_RUNS` could be also suitable here (and in `AbstractCPMessageTask` usages)?